lib/order.rb

```diff
@@ -0,0 +1,72 @@
+##################################
+require 'CSV'
+##################################
+
+class Order
+  attr_reader :id, :products, :customer, :fulfillment_status, :expected_total
+
+  def initialize(id, products, customer, fulfillment_status = :pending)
+    @id = id
+    @products = products
+    @customer = customer
+    @fulfillment_status = fulfillment_status
+    @expected_total = 0
+
+    status_options = [:pending, :paid, :processing, :shipped, :complete]
```

Comment:
Consider moving this error checking to before you assign the instance variables in the constructor.